src/LinearSolvers/iterativeWrapper.jl

```diff
@@ -24,6 +24,7 @@ Example:
 
 """
 type IterativeSolver<: AbstractSolver
+   AA   # matrix being solved
```

Comment:
In your DivSigGrad, after the A matrix is calculated, it is always stored.  When using the MUMPS solver, there is no reason to keep it in memory after it is factored.  It is only when using the iterative solver that you want to have access to A for sensitivity calculation.  This is the reason why I think it should be part of iterativeWrapper.